content/300-guides/050-database/900-turso.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Using Prisma with Turso'
+metaTitle: 'Using Prisma with Turso'
+metaDescription: 'Guide to using Prisma with Turso'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and Turso, explains the commonalities and differences between Turso and other database providers, and leads you through the process for configuring your application to integrate with Turso
+
+</TopBlock>
+
+## What is Turso?
+
+[Turso](https://turso.tech/) is an edge-hosted, distributed database that's based on [libSQL](https://turso.tech/libsql), an open-source and open-contribution fork of [SQLite](https://sqlite.org/), enabling you to bring data closer to your application and minimize query latency.
+
+## Commonalities with other database providers
```

Comment:
This section looks different compared to https://github.com/prisma/docs/pull/5397 (for Neon), is that intended?

Reply:
Yes. While it would be nice to have consistency, I think it would be nice to provide value in this section 🙂